examples/generated/test/swift/style/TextStyles.swift

```diff
@@ -22,7 +22,6 @@ public class TextStyles: UIView {
 
```

Comment:
Files in `examples/generated`  are generated by the Lona compiler, so we shouldn't lint them. Can we add `examples` to the list of directories `excluded` by SwiftLint?

Reply:
Yes sure 🔥